src/Sulu/Bundle/SnippetBundle/Tests/Functional/Controller/SnippetControllerTest.php

```diff
@@ -50,18 +50,19 @@ class SnippetControllerTest extends SuluTestCase
     public function setUp()
     {
         parent::setUp();
-        $this->documentManager = $this->getContainer()->get('sulu_document_manager.document_manager');
-        $this->phpcrSession = $this->getContainer()->get('sulu_test.doctrine_phpcr')->getConnection();
         $this->initPhpcr();
-        $this->loadFixtures();
         $this->client = $this->createAuthenticatedClient();
```

Comment:
Why can we do it like this in this Test, but have to put all of this stuff on static variables in the `NodeControllerTest`?

Reply:
Also missing an explanation here, probably connected to the comment above.

Reply:
because `$this->client = $this->createAuthenticatedClient();` is called here before binding the services to the class variables so the service has the new kernel in them and not a kernel which get destroyed.